FHIR-Converter/client/src/core/template-manager/acr-template-manager.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License in the project root for license information.
+ */
+
+import * as cp from 'child_process';
+import * as templateManagementConstants from '../common/constants/template-management';
+import { TemplateManagementError } from '../../core/common/errors/template-management-error';
+import { ITemplateManager } from './template-manager';
+
+export class AcrTemplateManager implements ITemplateManager {
+	private _orasExePath: string;
+	private _templateManagementExePath: string;
+	
+	constructor (templateManagementExePath = templateManagementConstants.DefaultTemplateManagementExePath, orasExePath = templateManagementConstants.DefaultOrasExePath) {
+		this._templateManagementExePath = templateManagementExePath;
+		this._orasExePath = orasExePath;
+	}
+
+	login(registryName: string) {
+		// Return cmd string to use terminal
+		return `${this._orasExePath} login ${registryName}`;
+	}
+
+	logout(registryName: string) {
+		try {
+			cp.execFileSync(this._orasExePath, ['logout', registryName]);
+			return 'Logout succeeded.';
+		} catch (err) {
+			throw new TemplateManagementError(err.stderr.toString());
+		}
+	}
+
+	pullTemplates(imageReference: string, outputFolder: string, force: boolean) {
+ 		try {
+			 const paramList = ['pull', imageReference, outputFolder];
```

Comment:
An extra space in the front of this sentence.

Reply:
Thanks, removed it.